CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 ### Added
```

Comment:
Update this to Changed instead? I think the other one is a change as well.

Reply:
oops yeah good point